server/src/services/person.service.ts

```diff
@@ -614,4 +631,21 @@ export class PersonService {
     }
     return person;
   }
+
+  private withOrientation(orientation: Orientation, { width, height }: ImageDimensions): ImageDimensions {
+    switch (orientation) {
+      case Orientation.Horizontal:
+      case Orientation.MirrorHorizontal:
+      case Orientation.Rotate180:
+      case Orientation.MirrorVertical: {
+        return { width, height };
+      }
+      case Orientation.MirrorHorizontalRotate270CW:
+      case Orientation.Rotate90CW:
+      case Orientation.MirrorHorizontalRotate90CW:
+      case Orientation.Rotate270CW: {
+        return { width: height, height: width };
```

Comment:
You don't have a fall through condition and the value of exif orientation is just stored as a string. It might be better to remove the top 4 cases and default to returning `{ width, height }` instead.

Reply:
Oh true, that's a good point.